src/state/routing/utils.ts

```diff
@@ -310,8 +310,17 @@ export async function transformRoutesToTrade(
   return { state: QuoteState.SUCCESS, trade: classicTrade }
 }
 
-function parseToken({ address, chainId, decimals, symbol }: ClassicQuoteData['route'][0][0]['tokenIn']): Token {
-  return new Token(chainId, address, parseInt(decimals.toString()), symbol)
+function parseToken({
+  address,
+  chainId,
+  decimals,
+  symbol,
+  buyFeeBps,
+  sellFeeBps,
+}: ClassicQuoteData['route'][0][0]['tokenIn']): Token {
+  const buyFeeBpsBN = buyFeeBps ? BigNumber.from(buyFeeBps) : undefined
+  const sellFeeBpsBN = sellFeeBps ? BigNumber.from(sellFeeBps) : undefined
+  return new Token(chainId, address, parseInt(decimals.toString()), symbol, undefined, false, buyFeeBpsBN, sellFeeBpsBN)
```

Comment:
we could do this unless we prefer undefined over 0% taxes
```suggestion
  buyFeeBps = 0,
  sellFeeBps = 0,
}: ClassicQuoteData['route'][0][0]['tokenIn']): Token {
  return new Token(chainId, address, parseInt(decimals.toString()), symbol, undefined, false, BigNumber.from(buyFeeBps), BigNumber.from(sellFeeBps))
```

Reply:
slightly more efficient to do `undefined` since there are default behaviors on router-sdk for `undefined` like [this](https://github.com/Uniswap/router-sdk/blob/main/src/entities/trade.ts#L171)